sechub-scan-product-sereco/src/test/java/com/mercedesbenz/sechub/domain/scan/product/sereco/SerecoProductResultTransformerTest.java

```diff
@@ -24,24 +28,93 @@
 import com.mercedesbenz.sechub.sereco.metadata.SerecoClassification;
 import com.mercedesbenz.sechub.sereco.metadata.SerecoCodeCallStackElement;
 import com.mercedesbenz.sechub.sereco.metadata.SerecoMetaData;
+import com.mercedesbenz.sechub.sereco.metadata.SerecoRevisionData;
 import com.mercedesbenz.sechub.sereco.metadata.SerecoSeverity;
+import com.mercedesbenz.sechub.sereco.metadata.SerecoVersionControl;
 import com.mercedesbenz.sechub.sereco.metadata.SerecoVulnerability;
 import com.mercedesbenz.sechub.sharedkernel.ProductIdentifier;
 
 public class SerecoProductResultTransformerTest {
 
     private SerecoProductResultTransformer transformerToTest;
 
-    @Before
-    public void before() {
+    @BeforeEach
+    void before() {
         transformerToTest = new SerecoProductResultTransformer();
         transformerToTest.falsePositiveMarker = mock(SerecoFalsePositiveMarker.class);
     }
 
     @Test
-    public void one_vulnerability_in_meta_results_in_one_finding() throws Exception {
+    void finding_revision_id_inside_sereco_is_transformed_to_sechub_result() throws Exception {
         /* prepare */
-        String converted = createMetaDataWithOneVulnerabilityFound();
+        String converted = createMetaDataWithOneVulnerabilityFoundWithVersionControlData(null, null, "finding-rev1");
+
+        /* execute */
+        ReportTransformationResult result = transformerToTest.transform(createProductResult(converted));
+
+        /* test */
+        Iterator<SecHubFinding> it = result.getResult().getFindings().iterator();
+        assertTrue(it.hasNext(), "no finding found!");
+
+        SecHubFinding finding = it.next();
+        Optional<SecHubRevisionData> revOpt = finding.getRevision();
+        assertTrue(revOpt.isPresent());
+        SecHubRevisionData revisionData = revOpt.get();
+        assertEquals("finding-rev1", revisionData.getId());
+    }
+
+    @Test
+    void version_control_inside_sereco_is_transformed_to_sechub_result() throws Exception {
+        /* prepare */
+        String converted = createMetaDataWithOneVulnerabilityFoundWithVersionControlData("revision1", "location1", null);
+
+        /* execute */
+        ReportTransformationResult result = transformerToTest.transform(createProductResult(converted));
+
+        /* test */
+        Optional<SecHubReportMetaData> metaDataOpt = result.getMetaData();
+        if (metaDataOpt.isEmpty()) {
+            fail("Did not found metadata");
```

Comment:
Found some typos, which might be copy paste, from the message in case the test fails. You can change them if you want to. They are optional changes to me.
```suggestion
            fail("Did not find metadata");
```